google/resumable_media/requests/download.py

```diff
@@ -19,6 +19,9 @@
 from google.resumable_media.requests import _helpers
 
 
+_SINGLE_GET_CHUNK_SIZE = 8192
```

Comment:
8192 is what gsutil uses, so it should be fine (assuming the underlying library pulling the data of the wire does this in a tight loop).  However, I haven't done an in-depth investigation/optimization of this constant, so it's possible there is room for improvement.